.gitignore

```diff
@@ -1,5 +1,6 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
+PyQt/__pycache__
```

Comment:
might not be necessary given the line before

Reply:
Line 3 was deleted